rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.80.0"
+channel = "nightly-2024-06-07"
```

Comment:
Rspack use nightly version of rust,but can switch to stable version in the future